src/gc/objecthandle.cpp

```diff
@@ -469,7 +469,7 @@ void CALLBACK ScanPointerForProfilerAndETW(_UNCHECKED_OBJECTREF *pObjRef, uintpt
 #endif
         break;
 
-#if defined(FEATURE_COMINTEROP) && !defined(FEATURE_REDHAWK)
+#if defined(FEATURE_COMINTEROP) && !defined(FEATURE_REDHAWK) && !defined(BUILD_AS_STANDALONE)
```

Comment:
COM interop is broken with standalone GC. Do you have a follow up bug to fix this?

Reply:
We have pretty light test coverage for COM interop in the open sourced tests today, so you might not have noticed the breakage if you have just run CoreCLR open sourced tests.

Reply:
I have only run the open source tests, and they all pass under local gc. But they also passed when it was compiled with FEATURE_REDHAWK so they don't test the corner cases of the gc.

I opened https://github.com/dotnet/coreclr/issues/17789